node/src/blockchain/blockchain_interface/blockchain_interface_web3/mod.rs

```diff
@@ -1188,7 +1233,7 @@ mod tests {
                 BlockMarker::Uninitialized,
                 &logger
             ),
-            61
+            BlockMarker::Value(61)
         );
         // Case 3: end_block_marker is Uninitialized, highest block in transactions is Uninitialized, start_block_marker is Value
```

Comment:
We said that maybe "no transactions retrieved" or something of that meaning would be better than the middle segment of this and also those other comments where it says "uninitialized".